### PR TITLE
Add nodiscard in GlCanvas

### PR DESCRIPTION
**Pull request by karupayun**
TODO after comments in https://github.com/google/orbit/pull/1639.

### CODE REVIEW

File: src/OrbitGl/GlCanvas.h [hunk 1]
```diff
@@ -89,23 +89,23 @@ class GlCanvas {
...
-  float GetWorldTopLeftX() const { return world_top_left_x_; }
-  float GetWorldTopLeftY() const { return world_top_left_y_; }
+  [[nodiscard]] float GetWorldWidth() const { return world_width_; }
+  [[nodiscard]] float GetWorldHeight() const { return world_height_; }
+  [[nodiscard]] float GetWorldMaxY() const { return world_max_y_; }
+  [[nodiscard]] float GetWorldTopLeftX() const { return world_top_left_x_; }
+  [[nodiscard]] float GetWorldTopLeftY() const { return world_top_left_y_; }
   virtual void UpdateWorldTopLeftY(float val) { world_top_left_y_ = val; }
 
   TextRenderer& GetTextRenderer() { return text_renderer_; }
```
Comment (dimitry-) [inline src/OrbitGl/GlCanvas.h:99]: here?

File: src/OrbitGl/GlCanvas.h [hunk 2]
```diff
@@ -89,23 +89,23 @@ class GlCanvas {
   virtual std::vector<std::string> GetContextMenu() { return std::vector<std::string>(); }
```
Comment (dimitry-) [inline src/OrbitGl/GlCanvas.h:89]: and probably here?

File: src/OrbitGl/GlCanvas.h (hunk 1 above)
Comment (karupayun) [reply src/OrbitGl/GlCanvas.h:99]: Thanks!

File: src/OrbitGl/GlCanvas.h (hunk 2 above)
Comment (karupayun) [reply src/OrbitGl/GlCanvas.h:89]: Thanks again, I also added a third one, sorry.

**Review by dimitry- — APPROVED** [review]

**Pull request merged**